### PR TITLE
Ruletest JSON for 22-28

### PR DESCRIPTION
**Pull request by jugonzal07**
First cut at rule test 22-28

### CODE REVIEW

**Comment by yunjoonjung-PNNL** [comment]
Juan, it seems like ```heat_rejections``` keys are missing. Could you check them out?

**Comment by jugonzal07** [comment]
> Juan, it seems like `heat_rejections` keys are missing. Could you check them out?

Thanks @yunjoonjung-PNNL. I have made the update.

File: rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section22/rule_22_28.json [hunk 1]
```diff
@@ -0,0 +1,415 @@
...
+                                "type": "CONDENSER",
+                                "cooling_or_condensing_design_and_control": {
+                                    "id": "CCDC 1",
+                                    "flow_control": "FIXED_FLOW"
+                                }
+                            }
+                        ],
+                        "heat_rejections": [
+                            {
+                                "id": "Heat Rejection 1",
```
Comment (yunjoonjung-PNNL) [inline rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section22/rule_22_28.json:158]: I believe ```Heat Rejection 1``` and ```Condensing Loop 1``` in line 148 must match. Could you update one of the values?

**Comment by weilixu** [comment]
@jugonzal07  @yunjoonjung-PNNL  This needs to addressed before merging 22-28 - otherwise, this rule seems good.

**Review by yunjoonjung-PNNL — APPROVED** [review]
Looks great to me. Thanks Juan!

**Pull request merged**